src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-link {
+  text-align: center;
+}
+
+a.is-active {
+  color: #00acdc;
+  text-decoration: underline;
+  position: relative;
+}
+
+.is-active::after {
+  content:"";
+  position: absolute;
+  top: 30px;
+  left: 0;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+nav {
+  height: 60px;
+  background-color: #fff;
+}
+
+.logo-list {
+  list-style: none;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  margin-left: 20px;
+}
```

Comment:
You've added a left margin to the `.nav__item` class. This will add extra space on the left side of the first item in the navigation. To avoid this, you could use the `:not(:first-child)` pseudo-class to select all items except the first one. For example: `.nav__item:not(:first-child) { margin-left: 20px; }`.